add guidelines.md

khareyash05 opened the pull request:
Fixes #58 
Added guidelines.md 
It will benefit even those who are beginners in open-source

siv2r commented on guidelines.md:
```diff
@@ -0,0 +1,91 @@
+
+Remember your upstream base branch should be master and source should be your feature branch. Click on create pull request and add a name to your pull request. You can also describe your feature.
+
+Awesome! You have made your first contribution. If you have any doubts please let me know in the comments.
```
You can remove the line "If you have any doubts please let me know in the comments"

siv2r commented on guidelines.md:
````diff
@@ -0,0 +1,91 @@
+***
+It can happen that your feature takes time to complete and other contributors are constantly pushing code. After completing the feature your feature branch should be rebase on latest changes to upstream master branch.
+
+```$ git checkout <feature-branch>```
````
write both the git commands under the same triple quotes sign.
For Example,
```
$git command1
$git command2
```

siv2r commented on guidelines.md:
````diff
@@ -0,0 +1,91 @@
+You have completed the feature, but you have made a number of commits which make less sense. You should squash your commits to make good commits.
+
+```$ git rebase -i HEAD~5```
+This will open an editor which will allow you to squash the commits.
````
This sentence continuous in the same line as the prev git command (when rendering). Start this in a new line.
This issue persists in many lines that come after a command. I have only mentioned a few in my comments.

siv2r commented on guidelines.md:
````diff
@@ -0,0 +1,91 @@
+This will create a new branch out of master branch. Now start working on the problem and commit your changes.
+
+```$ git add --all```
+```$ git commit -m "<commit message>"```
````
write both the git commands under the same triple quotes sign.

siv2r commented on guidelines.md:
````diff
@@ -0,0 +1,91 @@
+Normally, all repositories have a master branch which is considered to remain stable and all new features should be made in a separate branch and after completion merged into master branch. So we should create a new branch for our feature or bugfix and start working on the issue.
+
+```$ git checkout -b <feature-branch>```
+This will create a new branch out of master branch. Now start working on the problem and commit your changes.
````
This sentence continuous in the same line as the prev git command (when rendering). Start this in a new line.

siv2r commented on guidelines.md:
````diff
@@ -0,0 +1,91 @@
+```$ git checkout master```
+```$ git fetch upstream```
+```$ git merge upstream/master```
+```$ git push origin master```
````
write all the git commands under the same triple quotes sign as shown below.
 - start quotes
 - git command one
 - git command two
 - git command three
 - git command four
 - end quotes

siv2r commented:
Also, can you rename the file to `CreatePR.md` and add a link to this document in the `README.md` file (similar to the contributor's document)

khareyash05 commented:
@siv2r  I have updated the following and checked 
1. Added 2 or more lines of code in one block
2. Added a `<br>` after code so that they come to the next line
3. Renamed the file to CreatePR 
4. Added a link to the file in README

siv2r commented:
Amazing work!!! :tada: :tada:

khareyash05 commented:
Thank you @siv2r . Could you please add a GSSOC label and also a level label to the issue?
